Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'jekyll', '~>4.3.2'
+gem 'jekyll-sass-converter', '~>2.2.0'
```

Comment:
I've found 3.0 (what jekyll 4 depends on) to be easier to run across platforms. Any reason to pin to 2.x?

Reply:
Yes, I found that building this website with latest jekyll and sass converter doesn't work as the latest sass-converter has a bunch of breaking changes, hence I created https://github.com/hillhacks/website/issues/57 to fix this in future and pin dependencies with any working version for now.